assets/schema_input.json

```
@@ -10,17 +10,21 @@
             "sample": {
                 "type": "string",
                 "pattern": "^\\S+$",
-                "errorMessage": "Sample name must be provided and cannot contain spaces"
+                "errorMessage": "Sample name must be provided and cannot contain spaces",
+                "meta": ["sample"]
             },
             "datatype": {
                 "type": "string",
                 "pattern": "^\\S+$",
-                "errorMessage": "Data type, and must be one of: 'pacbio'"
+                "errorMessage": "Data type, and must be one of: 'pacbio'",
+                "meta": ["datatype"]
             },
             "datafile": {
                 "type": "string",
+                "exists": true,
                 "pattern": "^\\S+\\.(bam|cram)$",
```

```suggestion
                "pattern": "^\\S+\\.(bam|cram)$",
                "format": "file-path",
                "exists": true,
```

Can force a check to make sure the input file exists!